project/stac_api/models.py

```diff
@@ -1,28 +1,307 @@
 from django.contrib.gis.db import models
+from django.utils.translation import gettext_lazy as _
+from django.core.exceptions import ValidationError
+from django.contrib.postgres.fields import ArrayField
+import re
 
 # pylint: disable=fixme
 # TODO remove this pylint disable once this is done
 
 # st_geometry bbox ch
 BBOX_CH = 'POLYGON((2317000 913000,3057000 913000,3057000 1413000,2317000 1413000,2317000 913000))'
 
+# I allowed myself to make excessive use of comments below, as this is still work in progress.
+# all the comments can be deleted later on
+
+# For Collections and Items: No primary key will be defined, so that the auto-generated ones
+# will be used by Django. For assets, a primary key is defined as "PositiveBigInteger" due the
```

Comment:
as mentioned: default case in django is not to explicitly handle the primary key and leave this to django. In case `models.AutoField` is not large enough (large number of objects excpected), `id = models.BigAutoField(primary_key=True)` can be defined

Reply:
BigAutoField is used for asset_id, I adapted the comment